src/htmlParser.js

```diff
@@ -261,7 +261,7 @@ export default class HtmlParser {
     }
 
     if (options.fix) {
-      this._fixedReadToken = fixedReadTokenFactory(this, options);
+      this._fixedReadToken = fixedReadTokenFactory(this, options, () => {return this._readTokenImpl();});
```

Comment:
I removed the braces, those were indeed unnecessary. Just `this._readTokenImpl` won't work since it wouldn't be bound to the parser instance (and accesses the stream).